lib/Client.php

```diff
@@ -271,6 +284,46 @@ private function send($key, $value, $type, $sampleRate)
     }
 
     /**
+     * @param $tags
+     * @return string
+     */
+    private function generateTagsMessage($tags)
+    {
+      // Nothing to return if no tags
+      if (empty($tags)) {
+        return '';
+      }
+
+      // Normalize string inputs into an array
+      if (!is_array($tags)) {
+        $tags = [$tags];
```

Comment:
Based on [this link](https://help.datadoghq.com/hc/en-us/articles/206441345-Send-metrics-and-events-using-dogstatsd-and-the-shell), they accept valueless tags. I also modeled after the [tag handing logic](https://github.com/DataDog/php-datadogstatsd/blob/master/libraries/datadogstatsd.php#L150) in the datadog/php-datadogstatsd repo.